modules/apigee_edge_teams/src/Form/AddTeamMembersForm.php

```diff
@@ -136,6 +134,12 @@ public function buildForm(array $form, FormStateInterface $form_state, TeamInter
       '#options' => $role_options,
       '#multiple' => TRUE,
       '#required' => FALSE,
+      TeamRoleInterface::TEAM_MEMBER_ROLE => [
```

Comment:
This is what I would expect here if we would like to replace what Drupal core does.
```
    // Special handling for the inevitable "Authenticated user" role.
    $form['account']['roles'][RoleInterface::AUTHENTICATED_ID] = [
      '#default_value' => TRUE,
      '#disabled' => TRUE,
    ];
```

Reply:
:+1: